Source/Scene/PointPrimitive.js

```diff
@@ -0,0 +1,488 @@
+     * and rendered using a {@link PointPrimitiveCollection}.  A pointPrimitive is created and its initial
+     * properties are set by calling {@link PointPrimitiveCollection#add}.
+     *
+     * @alias PointPrimitive
+     *
+     * @performance Reading a property, e.g., {@link PointPrimitive#show}, is constant time.
+     * Assigning to a property is constant time but results in
+     * CPU to GPU traffic when {@link PointPrimitiveCollection#update} is called.  The per-pointPrimitive traffic is
+     * the same regardless of how many properties were updated.  If most pointPrimitives in a collection need to be
+     * updated, it may be more efficient to clear the collection with {@link PointPrimitiveCollection#removeAll}
+     * and add new pointPrimitives instead of modifying each one.
+     *
+     * @exception {DeveloperError} scaleByDistance.far must be greater than scaleByDistance.near
+     * @exception {DeveloperError} translucencyByDistance.far must be greater than translucencyByDistance.near
+     *
+     * @see PointPrimitiveCollection
+     * @see PointPrimitiveCollection#add
+     * @see Label
+     *
+     * @internalConstructor
+     *
+     * @demo {@link http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=PointPrimitives.html|Cesium Sandcastle PointPrimitive Demo}
+     */
+    var PointPrimitive = function(options, pointPrimitiveCollection) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        //>>includeStart('debug', pragmas.debug);
+        if (defined(options.scaleByDistance) && options.scaleByDistance.far <= options.scaleByDistance.near) {
+            throw new DeveloperError('scaleByDistance.far must be greater than scaleByDistance.near.');
+        }
+        if (defined(options.translucencyByDistance) && options.translucencyByDistance.far <= options.translucencyByDistance.near) {
+            throw new DeveloperError('translucencyByDistance.far must be greater than translucencyByDistance.near.');
+        }
+        //>>includeEnd('debug');
+
+        this._show = defaultValue(options.show, true);
+        this._position = Cartesian3.clone(defaultValue(options.position, Cartesian3.ZERO));
+        this._actualPosition = Cartesian3.clone(this._position); // For columbus view and 2D
+        this._color = Color.clone(defaultValue(options.color, Color.WHITE));
+        this._outlineColor = Color.clone(defaultValue(options.outlineColor, Color.TRANSPARENT));
+        this._outlineWidth = defaultValue(options.outlineWidth, 0.0);
+        this._pixelSize = defaultValue(options.pixelSize, 10.0);
+        this._scaleByDistance = options.scaleByDistance;
+        this._translucencyByDistance = options.translucencyByDistance;
+        this._id = options.id;
+        this._collection = defaultValue(options.collection, pointPrimitiveCollection);
+
+        this._pickId = undefined;
+        this._pickPrimitive = defaultValue(options._pickPrimitive, this);
```

Comment:
We don't need this.

This is for when a label is implemented as a bunch of billboards and we want the label to be returned when the billboard is picked.